WcaOnRails/db/migrate/20160502113530_add_preferred_events_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreferredEventsToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :preferred_event_ids, :text
```

Comment:
I get it, that's definitely better. How would you update the table when the user sends the form with preferred events?